### PR TITLE
github-actions: added parallelize client tests

### PR DESCRIPTION
**Pull request by Aleksey28**
[closes DevExpress/testcafe-private#221]

## Purpose
_Describe the problem you want to address or the feature you want to implement._

## Approach
_Describe how your changes address the issue or implement the desired functionality in as much detail as possible._

## References
https://github.com/DevExpress/testcafe-build-system/pull/22

## Pre-Merge TODO
- [ ] Write tests for your proposed changes
- [ ] Make sure that existing tests do not fail

### CODE REVIEW

**Comment by Aleksey28** [comment]
Merge only after https://github.com/DevExpress/testcafe-build-system/pull/22 and rewriting links to actions.

**Review by miherlosev — CHANGES_REQUESTED** [review]
Is it a development artifact?

File: .github/workflows/test-client.yml [hunk 1]
```diff
@@ -45,8 +54,23 @@ jobs:
       - run: ${{ inputs.test-script }}
         timeout-minutes: 60
 
-      - uses: DevExpress/testcafe-build-system/actions/set-status@main
+      - uses: Aleksey28/testcafe-build-system/actions/save-matrix-status@main
```
Comment (miherlosev) [inline .github/workflows/test-client.yml:57]: Is it a development artifact?

**Review by miherlosev — CHANGES_REQUESTED** [review]

**Review by miherlosev — CHANGES_REQUESTED** [review]
Also remove `sc-....log` files

File: .github/workflows/test-client.yml (hunk 1 above)
Comment (Aleksey28) [reply .github/workflows/test-client.yml]: It was temporary until https://github.com/DevExpress/testcafe-build-system/pull/22 wasn't merged.

File: .github/workflows/test-functional.yml [hunk 2]
```diff
@@ -157,44 +157,25 @@ jobs:
...
-          script: |
-            const statuses = ${{ toJSON(needs.test.outputs.*) }};
-
-            for (let i = 0; i < ${{ inputs.matrix-jobs-count }}; i++) {
-              if (statuses[i] !== 'success')
-                return 'failure';
-            }
 
-            return 'success';
+      - uses: Aleksey28/testcafe-build-system/actions/save-matrix-status@main
```
Comment (miherlosev) [inline .github/workflows/test-functional.yml:161]: Change to DevExpress/testcafe-build-system'

**Review by miherlosev — CHANGES_REQUESTED** [review]

**Review by miherlosev — APPROVED** [review]

**Pull request merged**